css-flexbox-practice.css

```diff
@@ -0,0 +1,102 @@
+.main {
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+.site {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+    max-width: 800px;
+}
+
+.navigation {
+    display: flex;
+    justify-content: space-between;
+}
+
+.nav-list {
+    list-style-type: none;
+    display: flex;
+}
+
+.list-item {
+    margin-left: 30px;
+}
+
+.section-1{
+    display: flex;
+    justify-content: space-between;
+    border: 1px solid black;
+    margin-bottom: 50px;
+}
+
+.section-1-left {
+    display: flex;
+    width: 25%;
+}
+
+.section-1-right {
+    width: 75%;
+    align-items: center;
+}
+
+.section-2 {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 50px;
+}
+
+.section-3 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: 50px;
+    max-width: 80%;
+}
+
+.box {
```

Comment:
This should be a flex container.
You are trying to move things around with padding, which is a no-no.
Do this with flex container, using justify-content, align-items, etc.

Reply:
I have made it a flex container and align-items: center;
This does give some room from the side of the box but not a lot. Should I give the box a width?